### PR TITLE
feat: add base network

### LINKED ISSUE #120: FRP: add base network
### Proposal

- add base support that is up in ydaemon

### Solve

This helps know what metadata isnt updated yet

### Thoughts

_No response_

### PR DESCRIPTION
**Pull request by 0xBasically**
## Description

close #120

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	"locale": "en-US",
 	"uri": "https://sync.yearn.farm/",
 	"og": "https://sync.yearn.farm/og.jpeg",
-	"twitter": "@iearnfinance",
+	"twitter": "@yearnfi",
```
Comment (karelianpie) [inline public/manifest.json]: Can we do this in another PR @0xBasically, that way we can get this in sooner

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
 		"lint": "eslint . --ext .js,.jsx,.ts,.tsx"
 	},
 	"dependencies": {
-		"@yearn-finance/web-lib": "^0.13.22",
+		"@yearn-finance/web-lib": "https://github.com/0xBasically/web-lib.git#add--base",
```
Comment (karelianpie) [inline package.json]: We can use the latest web-lib

Comment (0xBasically) [reply package.json]: ok
